web_development_101/installfest_first_rails_app.md

```diff
@@ -487,21 +487,15 @@ git commit -m "initial commit"
 
 #### Step 3.3: Hooking our local repository to the remote repository
 
-We need to tell git where to send the files, to do this we will need to create a remote repository and take the address it creates, and add it to git.
+At this point we have our files on our own computer and can track changes made to our file over time. But what if we want to share with another computer or have multiple people work on our project with us? For this we'll need a GitHub repository.
 
-First open [GitHub](https://github.com/) in the browser and sign in if you aren't already. Next click the `+` symbol next to your profile picture in the upper right and click `New Repostiory`. Then give the repository a name, and click the `Create Repository` button below the form. You can leave all the other options alone.
+First we'll open [GitHub.com](https://github.com/) in the browser and sign in (if you aren't already). Next, look for your profile picture in the upper right hand corner, click the + symbol next to it, and create a `new repository`. Give the repository a name (Maybe `my_first_rails_app`?), then click `Create Repository` below the form. For your first project you shouldn't need to mess with any of the other options, but they're good to read over for future projects!
 
-GitHub should have taken you to a page that has some commands listed.
+On your `Create Repository` page you'll see various commands listed. We know that we want to use SSH to connect to GitHub so make sure `SSH` has been selected. Look for the section that says "... or push an existing repository from the command line" and click the icon to the right of it to copy the text to your clipboard. Then paste it into the terminal we have open (Using CTRL + SHIFT + V in Linux or CTRL + V in WSL).
```

Comment:
Bold: **make sure `SSH` has been selected** maybe?

I can see people skimming this too quickly